autogluon/contrib/enas/enas_scheduler.py

```diff
@@ -126,12 +126,15 @@ def run(self):
                     graph = self.supernet.graph
                     graph.attr(rankdir='LR', size='8,3')
                     tbar.set_svg(graph._repr_svg_())
-                tbar.set_description('avg reward: {:.2f}'.format(self.baseline))
+                if self.baseline:
+                    tbar.set_description('avg reward: {:.2f}'.format(self.baseline))
                 idx += 1
             self.validation()
             self.save()
-            tq.set_description('epoch {}, val_acc: {:.2f}, avg reward: {:.2f}' \
-                        .format(epoch, self.val_acc, self.baseline))
+            msg = 'epoch {}, val_acc: {:.2f}'.format(epoch, self.val_acc)
+            if self.baseline:
+                msg += ', avg reward: {:.2f}'.format(self.baseline)
```

Comment:
have you tried this, will it becomes too long to fit in one line for progress bar?

Reply:
it's able to fit in one line as long as the terminal window isn't too small